README.md

```diff
@@ -244,6 +244,44 @@ Route::group(function() {
 })->middleware('cacheResponse:600');
 ```
 
+### Tagging Routes
+
+If the cache driver you configured supports tags, you can specify a list of tags when applying the middleware:
+
+```php
+// add a "snowflake" tag to this route
+Route::get('/my-special-snowflake', 'SnowflakeController@index')->middleware('cacheResponse:,snowflake');
```

Comment:
Is that `:,` correct?

Reply:
Based on the implementation in it's current state: yes. The first parameter passed to the middleware is always assumed to be the cache lifetime, so here we're neglecting to define a lifetime such that it reverts to the default as specified in the config.

Alternatively, we could attempt to parse the first argument. If we parse a non-zero integer, then we assume it's the lifetime, otherwise assume it's the tags. The only difficulty here is if one were to pass `0` as the cache lifetime, as this would be ignored. If that's not a concern (?), I'll make that change.

Unfortunately, I think we have to be at least a little strict with the order of the arguments. It would be ideal if we could simply allow parameters to be passed in any order, and figure out what to do based on what we parse. However, it's entirely possible for numeric cache tags to be used.